snippets/wifi-enterprise/wifi-enterprise_nrf.conf

```diff
@@ -0,0 +1,17 @@
+# Enable Wi-Fi enterprise mode
```

Comment:
you don't need to duplicate entire file, see https://docs.zephyrproject.org/latest/build/snippets/writing.html#board-specific-settings we can only add overrides here

Reply:
Corrected.